### ISSUE TITLE
Make a variable view

### ISSUE THREAD

**Issue by SimonBiggs** [SimonBiggs/scriptedforms#40]
Create another type of variable, the variable reference, or maybe variable view? It simply displays the variable within a card. It is useful for output, but also useful to link a particular variable to another live section. Example can be found within the electron measurement utility.

**Comment by robmarkcole** [comment]
I know my colleagues love the variable explorer in Spyder/Matlab. Not sure if that is what you are considering adding somehow  - [perhaps like this](http://volderette.de/jupyter-notebook-variable-explorer/)

**Comment by SimonBiggs** [comment]
Nah, that's not what I was going to do. That will exist within jupyterlab eventually and when I build a jupyterlab extension for this the variable explorer will then come for free.

This was simply a variable type which only displays variables and doesn't allow editing.
